tests/unit/dataset/dataset_test.py

```diff
@@ -351,7 +354,10 @@ def test_load_correct_preprocessed_gaze_dfs(dataset_configuration):
 
     expected_gaze_dfs = dataset_configuration['preprocessed_gaze_dfs']
     for result_gaze_df, expected_gaze_df in zip(dataset.gaze, expected_gaze_dfs):
-        assert_frame_equal(result_gaze_df.frame, expected_gaze_df.frame)
+        assert_frame_equal(
```

Comment:
You can just use `check_column_order=False` here

Reply:
done